spec/integration/post_index_spec.rb

```diff
@@ -0,0 +1,121 @@
+require 'rails_helper'
+
+RSpec.describe 'Post index page', type: :system do
+  let(:users) do
+    [
+      User.create(name: 'Daniel',
+                  photo: 'https://media.licdn.com/dms/image/D4E03AQEu5C9mJwO5SQ/profile-displayphoto-shrink_800' \
```

Comment:
- [ ] Please almost all your integration specs are failing as you can see below :point_down:  so make sure that you fix them :pray: 
  
  ![image](https://github.com/carreraprogrammer/rails-blog/assets/73854999/656762e6-b742-45d8-a277-544b7cd4f0a0)

Reply:
Hello @Lizdev-05, the error is because your version of Chrome is not compatible with the version that I'm using, please, check the error, is related to your chrome version


https://github.com/carreraprogrammer/rails-blog/assets/109081622/151a79b4-2f74-4f15-919b-991814902a16

Reply:
[Grabación de pantalla desde 25-07-23 22:21:52.webm](https://github.com/carreraprogrammer/rails-blog/assets/109081622/f54656de-11a3-463d-90a7-395fe7c931c5)